package.json

```diff
@@ -19,6 +19,7 @@
     "bootstrap": "^4.6.0",
     "bootstrap-vue": "^2.21.2",
     "core-js": "^3.15.1",
+    "node": "16.20.2",
```

Comment:
I don't think we need this here

Reply:
@dhaladitya108 I had node version 20 , it didn't work.